code/modules/economy/economy_misc_vr.dm

```diff
@@ -0,0 +1,13 @@
+/hook/startup/proc/pay_raise()
+	economic_species_modifier = list(
```

Comment:
Yea, why not just remove this functionality entirely? There's not much need for it.